yugabyte_cloudformation.yaml

```diff
@@ -460,8 +452,6 @@ Resources:
               command: !Sub 
                 - bash /home/ec2-user/create_universe.sh AWS ${AWS::Region} ${RF} ${Node0NetInt.PrimaryPrivateIpAddress} ${Node1NetInt.PrimaryPrivateIpAddress} ${Node2NetInt.PrimaryPrivateIpAddress} ${PublicSubnet0.AvailabilityZone} ${PublicSubnet1.AvailabilityZone} ${PublicSubnet2.AvailabilityZone} ${PublicSubnet2.AvailabilityZone}
                 - { RF: !Ref RFFactor }
-            03_Run_initdb: 
-              command: !If [YBVersion, '' , 'bash /home/ec2-user/init_ysql.sh']   
```

Comment:
If we select an older version of yugabyte, we need to run initdb command to initialize ysql. If removed it then we cant support older version using same cloudformation template.

Reply:
Well the check would fail even for newer versions, because it was not a sem ver check. For simplicity I have removed it, because going forward we won't need the init_ysql and we can pin our base version to 1.3.0.0 for cloudformation and terraform deployments. And not support older versions.